package.json

```
@@ -16,7 +16,10 @@
     "test.ci": "jest --maxWorkers=2",
     "test.coverage": "yarn test --coverage",
     "test.watch": "yarn test --watchAll",
+    "run.binary": "cd cli-binaries && ./$BINARY mock ../examples/$SPEC",
```

Prefer longer paths over cd in directories:
```suggestion
    "run.binary": "./cli-binaries/$BINARY mock ../examples/$SPEC",
```